controller/volume_controller.go

```diff
@@ -528,14 +528,29 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, e *longhorn
 		}
 		newVolume = true
 	}
-
 	if len(rs) == 0 {
 		// first time creation
 		if err = vc.replenishReplicas(v, e, rs); err != nil {
 			return err
 		}
 	}
 
+	allReplicaNew := true
+	for _, r := range rs {
+		if r.Spec.HealthyAt != "" {
+			allReplicaNew = false
+			break
+		}
+	}
+
+	if v.Spec.FromBackup != "" && v.Spec.Standby == false && allReplicaNew == true && v.Spec.RestorationCompleted == false {
```

Comment:
Why do we need `allReplicaNew`? 
Can we use `v.Spec.FromBackup != "" && v.Spec.RestorationCompleted == false && v.Spec.Standby == false && v.Status.State ==  types.VolumeStateDettached` instead?